framework/wazuh/exception.py

```diff
@@ -118,13 +118,16 @@ class WazuhException(Exception):
         1721: 'Remote upgrade is not available for this agent OS version',
         1722: 'Incorrect format for group_id. Characters supported  a-z, A-Z, 0-9, ., _ and -. Max length is 255',
         1723: 'Hash algorithm not available',
-        1724: 'Not a valid select field',
+        1724: {'message': 'Not a valid select field',
+               'remediation': 'Use a valid field'},
```

Comment:
This remediation tells nothing to the user

Reply:
I use an `extra_remediation` to specify the valid fields: https://github.com/wazuh/wazuh/blob/ec3ecc7bcaa8f7a1a6ae476722af968ff5aa5714/framework/wazuh/cluster/server.py#L164-L165